src/py/flwr/common/fedxgb.py

```diff
@@ -0,0 +1,145 @@
+# Copyright 2020 Adap GmbH. All Rights Reserved.
```

Comment:
I think this should be moved to somewhere else, probably `src/py/flwr/server/utils`, I'll get back to you to confirm that.